terracumber_config/tf_files/SUSEManager-Head-NUE.tf

```diff
@@ -12,12 +12,12 @@ variable "CUCUMBER_COMMAND" {
 
 variable "CUCUMBER_GITREPO" {
   type = string
-  default = "https://github.com/uyuni-project/uyuni.git"
+  default = "https://github.com/SUSE/spacewalk.git"
 }
 
 variable "CUCUMBER_BRANCH" {
   type = string
-  default = "master"
+  default = "Manager-4.2-qe-slesp3-to-sp4"
```

Comment:
`qe-slesp3-to-sp4` should be good enough in `uyuni` repo

Reply:
Fixed.